utils/appliance.py

```diff
@@ -1194,21 +1194,13 @@ def log_raise(exception_class, message):
                 if result.rc != 0:
                     log_raise(Exception, "Could not download VDDK")
 
-                # extract
-                log_callback('Extracting vddk')
-                status, out = client.run_command('tar xvf {}'.format(filename))
-                if status != 0:
-                    log_raise(Exception, "Error: Unknown format of the file:\n{}".format(out))
-
                 # install
                 log_callback('Installing vddk')
                 status, out = client.run_command(
-                    'vmware-vix-disklib-distrib/vmware-install.pl --default EULA_AGREED=yes')
+                    'yum -y install {}'.format(filename))
```

Comment:
Don't we need to include the .rpm extension? I'm making an assumption that there is an rpm in the .gz?